packages/inventory-compliance/src/PresentationalComponents/UnsupportedSSGVersion.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { Popover, Text } from '@patternfly/react-core';
+import { ExclamationTriangleIcon, OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
+
+const UnsupportedSSGVersion = ({ supported = true, ssgVersion }) => {
+    const bodyContent = 'This system is running an unsupported version of the SCAP Security Guide (SSG).' +
+        'This information is based on the last report uploaded for this system to the Compliance service.';
+    const supportedConfigsLink = 'https://access.redhat.com/documentation/en-us/red_hat_insights/2020-10/' +
+        'html/assessing_and_monitoring_security_policy_compliance_of_rhel_systems/' +
+        'compl-assess-overview-con#compl-assess-supported-configurations-con';
+    const footerContent = <a target='_blank' rel='noopener noreferrer' href={ supportedConfigsLink }>Supported SSG versions</a>;
+    const divStyle = {
+        display: 'block',
+        margin: '1rem -1.5rem -1.5rem -1.5rem'
+    };
```

Comment:
Using classes is not practical here. The selector would need to be very specific using patternfly classnames to properly address properties and redefine them. This also prevents the styling definitions possibly messing with other styles. There can be instances (and we had these in the past, i think) where class names change, or a definition gains more specificity and requires fixing a custom style. While this is totally expected, and there is not much we can do about that, it is easier to simply update CSS directly on the component than to re-evaluate the selector.

This style here is to (ab)use the patternfly alert styling, it is not a desired final solution, but accomplishes what we need for now. Since this might be a component used in the future in other applications, it could become a frontend-components component or even part of patternfly, which will allow us to completely remove these styles.

Reply:
Generally, use of classes is preferred over inline styling. It has technical and practical reasons, like separation of design and content. The selectors should be chosen in a way that they do not clash.

The final solution is very much coupled with the cards now. These margins tie it together, so the component cannot be used standalone or in another context. With a proper class usages, this can be prevented. For example the SystemPolicyCard can have its own custom class (eg. compliance-system-policy-card) and the component could have its own (eg. compliance-usupported-ssg-version). Margins can be defined for the context of a (system policy) card.

Reply:
> Generally, use of classes is preferred over inline styling.

Generally, yes, but not here. If this would be aimed to be a generic `CardAlert` component, to be used in any other context or the styling would not be based on patternfly styles, then yes, what you are describing would be desirable. But it is not. It is meant to work specifically just here. 

> The selectors should be chosen in a way that they do not clash.

This is not the only thing to consider. The selector also needs to be very specific to redefine certain properties. Especially padding and margin are not easy to catch, since the styling for the `Alert` is based on many other styles, that could hinder a later imported style to be effective.